api/package.json

```diff
@@ -68,7 +68,7 @@
     "newrelic": "8.11.1",
     "pg": "8.7.3",
     "sanitize-html": "2.7.0",
-    "sharp": "0.30.5",
+    "sharp": ">=0.30.5",
```

Comment:
can you set this to specifically 0.30.5? we are trying to lock down all the dependencies to specific versions